### ISSUE TITLE
[BUG]  toolkit:Popup ignores dimensions

### ISSUE THREAD

**Issue by engalbertobortolato** [CommunityToolkit/Maui#1812]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [X] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

toolkit:Popup or VerticalStackLayout ignores Height and Width

### Expected Behavior

A PopUp width 3000 pixel height and 3000 pixel width

### Steps To Reproduce

1.  Open and run the solutution
2.  Press "Click me" button
3.  See the PopUp

### Link to public reproduction project repository

https://github.com/engalbertobortolato/test-popup

### Environment

```markdown
- .NET MAUI CommunityToolkit: 8.0.1
- OS: Windows 11 Pro 23H2 
- .NET MAUI: 8.0.20
```

**Comment by Elmigo** [comment]
I'm having the same issue and tried downgrading. Doesn't help, even going as far back as MAUI Community Toolkit 6.1.0 and it still ignores widths, but it used to work until recently. Might be something else and not necessarily the MCT popup causing it.

**Comment by Elmigo** [comment]
I think I'm having the same issue and tried downgrading MCT. Even as far back as 6.1.0 the issue persists. All widths and heights are being ignored within the Popup view and on the Popup view itself. Might be something else causing it because older versions used to work until recently.

**Comment by asi-evin** [comment]
Try setting "HeightRequest" and "WidthRequest" instead of "Height" and "Width".

**Comment by bijington** [comment]
@engalbertobortolato is this issue resolved?

**Comment by engalbertobortolato** [comment]
> @engalbertobortolato is this issue resolved?

(sorry for my english)
is not resolved using Height and Width

![image](https://github.com/CommunityToolkit/Maui/assets/141247430/9a861987-fb89-4370-8731-68211aa71c4b)
![image](https://github.com/CommunityToolkit/Maui/assets/141247430/e49c7153-e56f-4b37-adf0-59a604583551)

But if I using HeightRequest and WidthRequest the popup take the dimension

**Comment by asi-evin** [comment]
Height and Width are read-only values (or atleast they should be as they behave that way). You should never be setting them, only fetching them when you need to know the actual dimensions of an element (i.e. Binding to them, or calculating stuff based on their dimensions in code behind).

TLDR: You should not be setting Width/Height directly, only setting Width/Height Requests.

**Comment by brminnick** [comment]
@engalbertobortolato Could you please confirm whether setting `HeightRequest` and `WidthRequest` resolves your issue?

**Comment by bdovaz** [comment]
This problem still exists even on the example on this repo, my other issue was ignored and automatically closed...

https://github.com/CommunityToolkit/Maui/issues/1854

**Comment by bijington** [comment]
This problem appears to be that the developer was using the `Width` and `Height` properties instead of `WidthRequest` and `HeightRequest` properties
